apps/anon-message-client/src/app/page.tsx

```diff
@@ -75,7 +82,8 @@ async function requestProof(
     fieldsToReveal: {
       argumentType: ArgumentTypeName.ToggleList,
       value: {},
-      userProvided: false
+      userProvided: false,
+      hideIcon: true
```

Comment:
why is this in `fieldsToReveal`?

Reply:
removed, was something accidentally copied over from `generateProofUrl` in `TelegramHelpers`.

it doesn't do anything